provisioning/device/src/TransportSettings/ProvisioningClientTransportSettings.cs

```diff
@@ -90,6 +90,7 @@ public abstract class ProvisioningClientTransportSettings
 
         // The default remote certificate validation callback. It returns false if any SSL level exceptions occurred
         // during the handshake.
+        [ExcludeFromCodeCoverage]
```

Comment:
Why exclude?

Reply:
Why private methods? They include code that is exercised as part of our SDK running, and could have bugs.

We can't unit test it a private method but we always have the option of making it internal or pulling out business logic from a class and making it an internal class that can be exercised independently.

Reply:
That's true